### ISSUE TITLE
508 Potential Performance Impact, noted during accessibility audits [PERFORMANCE, SCREENREADER] 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#4404]
## Description

While conducting accessibility audits, I was unable to access the "Skip to Content" links. A quick performance audit indicated a Time To Interactive (TTI) of 12–20 seconds, which may be the issue.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to skip the header area for various pages of a website so that I don't have to review all of that just to read the content of each page.

## Environment

* Operating System: Mac, Windows
* Browser: Safari, Firefox (respectively)
* Screenreading device: VoiceOver, NVDA (respectively)
* Server destination: see individual tickets

## Steps to Recreate

See individual tickets within this Epic.

## Possible Fixes (optional)

If the performance teams audit reveals this is in fact the issue, recommend considering:
* moving non-critical scripts to the footer
* adjust design for pages so that the areas that require the slower APIs are not needed immediately, by _**perhaps**_ explaining the page, _then_ having the content that needs that data appear (as an example of a potential redesign aimed at making the page _feel faster_ since these APIs constrain potential design at this time)
* change from Source Sans to Public Sans for VA.gov sitewide (Public Sans is 701k for 18 different styles, whereas Source Sans Pro is 4.33B for 12 styles, and we also load Bitter and Font Awesome)

## WCAG or Vendor Guidance (optional)

* [Marcy Sutton, Performance Impact on Accessibility](https://marcysutton.com/accessibility-and-performance)
* [Tammy Everts, When Design Best Practices Become Business Worst Practices](https://uxmag.com/articles/when-design-best-practices-become-performance-worst-practices)

**Comment by FranECross** [comment]
Closing old ticket; code has changed enough that this may be obsolete, and if not, then the DST will address when they componentize.
